tool-data/shared/ucsc/manual_builds.txt.sample

```diff
@@ -1,708 +1,708 @@
-phiX	phiX174	phiX=5386

```

Comment:
This is an externally downloaded file as well - I don't think we should modify it.

Reply:
Hmm I would have thought this was a safe change, but I'll trust your intuition and remove it just in case. It just means we will have a few files committed to our repo with CRLF line endings which isn't a huge deal as long as the files aren't altered often. Since this file hasn't been touched in years, I won't add a gitattribute rule for this file.

Reply:
PR had been updated with removal of this file